lib/router/package.json

```diff
@@ -42,15 +42,15 @@
   "dependencies": {
     "@storybook/client-logger": "6.5.0-rc.1",
     "core-js": "^3.8.2",
+    "memoizerific": "^1.11.3",
+    "qs": "^6.10.0",
```

Comment:
Technically qs wasn't erroring for me but https://github.com/storybookjs/storybook/pull/18015 moved it to dependencies